migrations/upgrades/20191128055729_update_directus_files.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+
+use Phinx\Migration\AbstractMigration;
+use function \Directus\get_random_string;
+
+class UpdateDirectusFiles extends AbstractMigration
+{
+    public function change()
+    {
+        $fieldsTable = $this->table('directus_fields');
+        $filesTable = $this->table('directus_files');
+        
+        if($this->checkFieldExist('directus_files', 'filename')){
+            $this->execute(\Directus\phinx_update(
+                $this->getAdapter(),
+                'directus_fields',
+                [
+                    'field' => 'filename_disk'
+                ],
+                ['collection' => 'directus_files', 'field' => 'filename']
+            ));
+        }
+
+        if (!$filesTable->hasColumn('filename_disk')) {
+            $filesTable->renameColumn('filename', 'filename_disk');
+        }
+    
+        if (!$filesTable->hasColumn('filename_download')) {
+            $filesTable->addColumn('filename_download', 'string', [
+                'limit' => 255,
+                'null' => true,
```

Comment:
This field can't be null. You can't download a file with `null` being the filename